### PR TITLE
[typo](docs) Modify parameter description

### PR DESCRIPTION
**Pull request by yuanyuan8983**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by zy-kkk — DISMISSED** [review]

**Comment by zy-kkk** [comment]
run buildall

File: docs/en/docs/advanced/variables.md [hunk 1]
```diff
@@ -640,7 +640,7 @@ Translated with www.DeepL.com/Translator (free version)
 
     <version since="1.2.0"></version>
 
-    Use a fixed replica to query. If use_fix_replica is 1, the smallest one is used, if use_fix_replica is 2, the second smallest one is used, and so on. The default value is -1, which means it is not enabled.
+    Use a fixed replica to search. replica starts with 0 and if use_fix_replica is 0, the smallest is used, if use_fix_replica is 1, the second smallest is used, and so on. The default value is -1, indicating that the function is disabled.
```
Comment (dataroaring) [inline docs/en/docs/advanced/variables.md:643]: search is not a better candidate than query.

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Review by LemonLiTree — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Review by zy-kkk — APPROVED** [review]

**Comment by zy-kkk** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Review by LemonLiTree — APPROVED** [review]

**Pull request merged**